src/server/middleware/path/path.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package path
+
+import (
+	"net/http"
+	"net/url"
+	"regexp"
+
+	"github.com/goharbor/harbor/src/server/middleware"
+)
+
+var (
+	defaultRegexps = []*regexp.Regexp{
+		regexp.MustCompile(`^/api/v2.0/projects/.*/repositories/(.*)/artifacts/?$`),
```

Comment:
How about replace the `2.0` with regexp as the version will bump up and we main maintain multiple versions of API